Assets/Scripts/trial/AbstractTrial.cs

```diff
@@ -58,6 +58,20 @@ protected AbstractTrial(int blockId, int trialId)
             {
                 maze = DataSingleton.GetData().MazesDictionary[Value.MazeName];
             }
+            // If the user hasn't set a MazeName we want to set the Maze to be
+            // unobtrusive, So the ground generates but nothing else.
+            else
+            {
+                maze = new Maze();
+                maze.WallHeight = 0;
+                maze.WallColor = "1B5E20";
+                maze.Sides = 4;
+                maze.GroundTileSides = 0;
+                maze.GroundTileSize = 0;
+                maze.GroundColor = null;
+                maze.Radius = 4;
+                maze.Position = new List<float> { 0, 0 };
```

Comment:
This could probably be done a lot cleaner if we wanted to refactor some stuff. This seems to work for now.

Reply:
Should probably be a builder

Reply:
Hey @hongyu-wang ! Wow, this is the first comment I've seen on an OpenMaze pull request in a long time 😄 .

I just updated to the shorter constructor syntax at least so this isn't _as_ ugly. 

Anyway, to address your comment: I guess we do have a few different types of mazes now don't we? It probably does make sense to actually have a Maze builder.

Unfortunately, it's kind of out of scope for this task. More so if you consider that to build a hard-coded Maze object here in the first place is a hack.

In any case, I appreciate the feedback. As far as I know, the intention is to hold off on improving the code base further as the next version may initiate/require a significant re-write.

In any case, your feedback is much appreciated.